docs/docs-content/clusters/edge/edge-host-management/export-cluster-definition.md

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_label: "Export Cluster Definition"
+title: "Export Cluster Definition"
+description: "Instructions for exporting cluster definition."
+hide_table_of_contents: false
+sidebar_position: 32
+tags: ["edge"]
+---
+
+You can export cluster definitions from a cluster in Palette and use the definition to provision a cluster in an Edge
+host. A cluster definition contains one or more cluster profiles, including the profile variables used in the profiles.
+
+:::preview
+
+:::
+
+## Prerequisites
+
+- At least one cluster profile in Palette.
+
+- A Palette API key.
```

Comment:
We can also link to the Create Palette API key guide. 
https://docs.spectrocloud.com/user-management/authentication/api-key/create-api-key